packages/entity-browser/src/input.js

```diff
@@ -0,0 +1,44 @@
+import {consoleLogger} from 'tocco-util'
+
+import {setEntityName, setFormBase, setAppId, setScrollBehaviour} from './modules/entityBrowser/actions'
+
+const isDefined = value => value !== undefined
+
+export const getDispatchActions = input =>
+  actionSettings.reduce((acc, actionSetting) => {
+    if (isDefined(input[actionSetting.name])) {
+      acc.push(actionSetting.action(...actionSetting.argsFactory(input)))
+    } else if (actionSetting.mandatory) {
+      consoleLogger.logError(`EntityBrowser: Mandatory field '${actionSetting.name}' not set in input`)
+    }
+    return acc
+  }, [])
+
+const actionSettings = [
+  {
+    name: 'entityName',
+    action: setEntityName,
+    mandatory: true,
+    argsFactory: input => [input.entityName]
+  },
+  {
+    name: 'entityName',
+    action: setFormBase,
+    argsFactory: input => [input.formBase || input.entityName]
+  },
+  {
+    name: 'formBase',
+    action: setFormBase,
+    argsFactory: input => [input.formBase]
+  },
```

Comment:
block starting on line 29 unnecessary? already handled in block on line 24? `entityName` is a mandatory prop, so the block on line 24 will always be executed an do everything that's needed for formBase, doesn't it?

Reply:
hm what happen if just `formBase` has changed without `entityName`?

Reply:
yeah, you're right. 🙈  nevermind